airbyte-container-orchestrator/Dockerfile

```diff
@@ -1,31 +1,31 @@
 ARG JDK_VERSION=19-slim-bullseye
 ARG JDK_IMAGE=openjdk:${JDK_VERSION}
-FROM ${JDK_IMAGE} AS sync-attempt
+FROM ${JDK_IMAGE} AS worker
```

Comment:
nit: should this be orchestrator for now until we come up with a better name?

Reply:
done